src/components/modal/AposModal.vue

```diff
@@ -101,11 +101,11 @@ export default {
     right: $spacing-double;
     bottom: $spacing-double;
     left: $spacing-double;
+    height: calc(100vh - #{$spacing-double * 2});
```

Comment:
This isn't critical since it's fixed position (and needs to be different for slide out anyway).